src/lib/storage/table.hpp

```diff
@@ -11,10 +11,20 @@ class TableStatistics;
 // A table is partitioned horizontally into a number of chunks
 class Table : private Noncopyable {
  public:
+  // TODO(team): Use this in other places.
+  struct ColumnDefinitionStruct {
+    std::string column_name;
+    std::string column_type;
+    bool column_nullable;
```

Comment:
Since this is a `ColumnDefinition`, I think it is clear that these refer to the column
```suggestion
    std::string name;
    std::string type;
    bool is_nullable;
```